crates/data_components/src/dynamodb/table_schema.rs

```diff
@@ -55,6 +58,10 @@ impl DynamoDBTableSchema {
         &self.table_schema
     }
 
+    pub fn time_format(&self) -> String {
```

Comment:
Could this be an `Arc<str>` instead to avoid unnecessary string cloning?

Reply:
Done